frontend-react/src/components/EditableCompare.tsx

```diff
@@ -207,6 +219,8 @@ export const EditableCompare = forwardRef(
                                 onBlur={(e) => {
                                     onBlurHandler(e?.target?.value || "");
```

Comment:
This makes our new button irrelevant, no? Perhaps instead of the button and the `onBlur`, could we debounce an `onChange` event so that it'll only diff after an idle of like 3 seconds? The UX as is feels a bit odd; like, halfway between trying to be automatic and manual.

Reply:
Sure, that's a fair point!  For context, this came out of a discussion between Penny, Tom, and me [in this pull request](https://github.com/CDCgov/prime-reportstream/pull/7639).  I can update this so that the blur doesn't handle the automatic reformatting/highlighting, but I think a timed update -- even with a debounce -- could introduce more confusion unless we had helper text saying when the update would be happening.  Plus, the user will lose their cursor positioning if we automatically reformat when the textarea has focus.

Reply:
I would't go that route. If you want to remove the onBlur then ensure that the Save is disabled anytime the textarea is changed so that the user has to click on the "Check syntax" button.

Reply:
Yeah, after reading the issues with the old UX, definitely more in favor of losing `onBlur` and disabling `Save` until they `Check Syntax`! 💯

Reply:
Thanks, everyone!  I just pushed up a commit here to only enable the 'Save' button if the 'Check syntax' shows that it's valid JSON: https://github.com/CDCgov/prime-reportstream/pull/7713/commits/f58b30724999df81bf11614f398e907531f53e8e

![2022-12-19 14 23 58](https://user-images.githubusercontent.com/2421042/208504385-b4a2f9aa-f39e-40de-9b7a-8e9337180e60.gif)

Let me know what y'all think! 🙏

Reply:
Oh, duh -- need to update the new tests!  Feel free to take a look at the code, and I'll push up the updated tests in the meantime.